packages/interbit/src/scripts/create.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs-extra')
+const path = require('path')
+const promisify = require('util').promisify
+const exec = promisify(require('child_process').exec)
+
+const TEMPLATE_VERSION = '0.4.27'
+
+const create = async options => {
+  const { appName, location } = options
+
+  process.chdir(location)
+  const newAppDir = path.join(location, appName)
+
+  console.log(`interbit create: Creating ${appName} in ${newAppDir}`)
+
+  if (!appName) {
+    console.error('interbit create: appName must be provided')
+    process.exit(1)
+  }
+
+  if (!appName.match(/^([a-z0-9-]+)+$/)) {
+    console.error(
+      'interbit create: app name must match regex /^(app|interbit)(-[a-z0-9]+)+$/'
+    )
+    process.exit(1)
+  }
+
+  if (fs.existsSync(newAppDir)) {
+    console.error(
+      `interbit create: Can not create app. Something already exists at location ${newAppDir}`
+    )
+    process.exit(1)
```

Comment:
Can we throw and just process exist in one place?  Makes this more reusable and you can have a general catch for that :)